src/Common/nunit/Options.cs

```diff
@@ -533,6 +546,17 @@ public OptionSet (Converter<string, string> localizer)
         public Converter<string, string> MessageLocalizer {
             get {return localizer;}
         }
+#else
+        string localizer(string msg)
```

Comment:
There is no Convert class in DNX and we don't use the localizer, so creating straight pass-through methods for this and the MessageLocalizer is the equivalent to the code in the default constructor that we use elsewhere.